packages/neos-ts-interfaces/src/index.ts

```diff
@@ -142,7 +142,40 @@ export interface PropertyConfiguration {
         [propName: string]: ValidatorConfiguration | undefined;
     };
 }
-
+export interface ReferencesConfiguration {
+    type?: string;
+    ui?: {
+        label?: string;
+        reloadIfChanged?: boolean;
+        inline?: {
+            editor?: string;
+            editorOptions?: {
+                [propName: string]: any;
+            };
+        }
+        inlineEditable?: boolean;
+        inspector?: {
+            hidden?: boolean;
+            defaultValue?: string;
+            editor?: string;
+            editorOptions?: {
+                [propName: string]: any;
+            }
+            group?: string;
+            position?: number | string;
+        };
+        help?: {
+            message?: string;
+            thumbnail?: string;
+        };
+    };
+    validation?: {
+        [propName: string]: ValidatorConfiguration | undefined;
+    };
+    properties?: {
```

Comment:
is this a new feature ✨Native support in the Neos ui?? :D Hurray

Reply:
Well, rather the preparation for using edge properties 😄